app/src/main/java/com/github/gotify/messages/ListMessageAdapter.java

```diff
@@ -157,6 +150,23 @@ void updateDate() {
             }
             date.setText(text);
         }
+
+        private void enableCopyToClipboard() {
```

Comment:
Currently, it doesn't work when long clicking on the text of a message, then the native text select of android will be triggered. I don't know if that is a problem, because it is maybe useful to have this functionality, what do you think?

Reply:
I added the OnLongClickListener to the ViewHolder because if it's added to the TextView, the area to press is reduce with short messages.
Right now, if you long click any free area, the text get copied.
I thought the same about native android text selection, could be useful for some one.